extensions/amp-story/1.0/page-advancement.js

```diff
@@ -560,6 +571,30 @@ export class ManualAdvancement extends AdvancementConfig {
     );
   }
 
+  /**
+   * Checks if click target is too large on the page and preventing navigation.
+   * If yes, the link is ignored & logged.
+   * @param {!Element} target
+   * @param {!ClientRect} pageRect
+   * @return {boolean}
+   * @private
+   */
+  isTooLargeOnPage_(target, pageRect) {
+    const targetRect = target./*OK*/ getBoundingClientRect();
+    if (
+      (targetRect.height * targetRect.width) /
+        (pageRect.width * pageRect.height) >=
+      MAX_LINK_SCREEN_PERCENT
+    ) {
+      user().error(
+        'AMP-STORY-PAGE',
+        'Link was too large; skipped for navigation. For more information, see https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md#links-in-amp-story-grid-layer'
```

Comment:
Linking to the github issue might be enough context, and has the benefit of linking to the code. If you want to move forward with the documentation that's great too, one nit though:

> Links that are too far to the left or right edge of a story page will be ignored for navigation.

We should make it clear that only the part that's by the edge will be ignored and yield to navigation

Reply:
Ah I like that. I left the clarification in the docs though since I think they are relevant caveats when we say that links are supported.